src/util/client.js

```diff
@@ -0,0 +1,10 @@
+import { ApolloClient } from 'apollo-client';
+import { HttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+const client = new ApolloClient({
+  link: new HttpLink({ uri: process.env.REACT_APP_API_ENDPOINT }),
+  cache: new InMemoryCache(),
```

Comment:
@fragosti nice! do you know how we could configure the cache settings?

Reply:
https://www.apollographql.com/docs/react/basics/caching.html